src/omv/modules/py_sensor.c

```diff
@@ -336,6 +336,10 @@ static mp_obj_t py_sensor_set_windowing(uint n_args, const mp_obj_t *args)
 
 static mp_obj_t py_sensor_get_windowing()
 {
+    if (sensor.framesize == FRAMESIZE_INVALID) {
+        mp_raise_msg(&mp_type_ValueError, MP_ERROR_TEXT("Frame size not set yet!"));
+    }
+
```

Comment:
```suggestion
```
Doesn't look related, if not please send in another PR.

Reply:
Moved to other PR.